packages/bits/src/lib/wizard-v2/wizard-step-header/wizard-step-header.component.ts

```diff
@@ -101,15 +134,32 @@ export class WizardStepHeaderComponent extends CdkStepHeader implements AfterVie
         return returnedState;
     }
 
+    private getIconsForStates(icons: IWizardIcons, customIcons: IWizardIcons): IWizardIcons {
+        const stepIcons = icons;
+        if (customIcons) {
+            Object.keys(customIcons).forEach(icon => {
+                stepIcons[icon] = customIcons[icon];
+            });
+        }
+
+        return stepIcons
+    }
+
     _getDefaultIconForState(state: StepState): string {
         let returnedState: StepState = state;
 
-        switch (state) {
-            case "number":
-            case "done": returnedState = "step-complete"; break;
-            case "edit": returnedState = "edit"; break;
-            case "error": returnedState = "warning"; break;
-            default:
+        if (state === "number") {
+            returnedState = this.stepIcon || this.icons.initial;
```

Comment:
I think it would be better to incorporate the value of `this.stepIcon` directly into `this.icons` to have a single source of truth for the icons.